pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
 python = ">=3.8.0,<4"
 datadog = ">=0.51.0,<1.0.0"
 wrapt = "^1.11.2"
-ddtrace = ">=2.20.0"
+ddtrace = "==2.21.3"
```

Comment:
Hmm, one concern here is that this change will also impact customers not using the layer. 

I'm wondering if instead of changing the compatibility requirements here, we should just force an install of the specific ddtrace version in the Dockerfile.

Reply:
So basically, here https://github.com/DataDog/datadog-lambda-python/blob/b64be3b098faf2743af1a1009f0c414f7fae8f33/Dockerfile#L17, we'd change this to something like

```Dockerfile
RUN pip install . ddtrace==2.21.3 -t ./python/lib/$runtime/site-packages
```

Reply:
Actually, I'm fine with however you wanna handle this @joeyzhao2018 since we're just gonna revert it back right away afterwards.

Reply:
The docker file is a better place i agree.